src/io/memory_block_io.h

```diff
@@ -147,14 +153,16 @@ MemoryBlockIO::ReadImpl(uint8_t* data, uint64_t size, uint64_t offset) const {
 }
 
 const uint8_t*
-MemoryBlockIO::ReadImpl(uint64_t size, uint64_t offset) const {
+MemoryBlockIO::ReadImpl(uint64_t size, uint64_t offset, bool& need_release) const {
     if (check_valid_offset(size + offset)) {
         if (check_in_one_block(offset, size + offset)) {
```

Comment:
unify name style: checkValidOffset / checkInOneBlock (like memory_io.h:112)

Reply:
we use under line style for private function. the memory_io.h will be fixed by other pull request